### PR TITLE
Improve and document `DuctRegistry`, remove `omniduct.session`, add `about()` functionality and add new logo.

### PR DESCRIPTION
**Pull request by matthewwardrop**
This adds documentation for the last significant undocumented piece of `omniduct`, the `DuctRegistry`. This registry has gone largely unchanged since its inception, but has never been super cohesive. This PR makes some breaking changes to the API ahead of the 1.0.0 release.

It adds an `about()` method, as well as machinery for downstream users of `omniduct` to do the same (including "endorsement" of omniduct and other projects).

It also removes the `omniduct.session` module that, while it seemed a good idea at the time, doesn't seem to offer much value.

Lastly, and of course most importantly, it adds the new logo (with minor changes from #85).

@danfrankj See any issues here?

### CODE REVIEW

File: omniduct/registry.py [hunk 1]
```diff
@@ -66,59 +137,172 @@ def __getitem__(self, name):
...
-        return self
+        Raises:
+            DuctNotFound: If no `Duct` can be found for requested name and/or
+                type.
+        """
+        if kind and not isinstance(kind, Duct.Type):
+            kind = Duct.Type(kind)
+        if name not in self._registry:
+            raise DuctNotFound(name)
+        r = self._registry[name]
```
Comment (danfrankj) [inline omniduct/registry.py]: consider expanding r->duct

Comment (matthewwardrop) [reply omniduct/registry.py]: Done.

File: omniduct/_version.py [hunk 2]
```diff
@@ -1,11 +1,13 @@
 import os
 import sys
 
-__all__ = ['__author__', '__author_email__', '__version__']
+__all__ = ['__author__', '__author_email__', '__version__', '__logo__', '__docs_url__']
 
 __author__ = "Matthew Wardrop, Dan Frank"
-__author_email__ = "matthew.wardrop@airbnb.com, dan.frank@airbnb.com"
+__author_email__ = "mpwardrop@gmail.com, dan.frank@airbnb.com"
```
Comment (matthewwardrop) [inline omniduct/_version.py]: @danfrankj You might want your email changed here too.

Comment (danfrankj) [reply omniduct/_version.py]: ah yes - would you put danfrankj@gmail.com here ?

Comment (danfrankj) [inline omniduct/_version.py]:
```suggestion
__author_email__ = "mpwardrop@gmail.com, danfrankj@gmail.com"
```

Comment (matthewwardrop) [reply omniduct/_version.py]: Done

Comment (danfrankj) [reply omniduct/_version.py]: trying out this new github feature ...

Comment (matthewwardrop) [reply omniduct/_version.py]: Oh... nice. I didn't know you could do that. Looks like that's a new feature.

File: omniduct/_version.py [hunk 3]
```diff
@@ -1,11 +1,13 @@
...
 import sys
 
-__all__ = ['__author__', '__author_email__', '__version__']
+__all__ = ['__author__', '__author_email__', '__version__', '__logo__', '__docs_url__']
 
 __author__ = "Matthew Wardrop, Dan Frank"
-__author_email__ = "matthew.wardrop@airbnb.com, dan.frank@airbnb.com"
+__author_email__ = "mpwardrop@gmail.com, danfrankj@gmail.com"
 __version__ = "0.9.6"
+__logo__ = os.path.join(os.path.dirname(__file__), 'logo.png') if '__file__' in globals() else None
```
Comment (danfrankj) [inline omniduct/_version.py]: :)

**Pull request merged**